cli/src/main/kotlin/commands/AdvisorCommand.kt

```diff
@@ -136,6 +136,16 @@ class AdvisorCommand : OrtCommand(
             throw ProgramResult(1)
         }
 
+        with(advisorResults.getVulnerabilities()) {
+            val totalPackageCount = ortResultOutput.getPackages().size
```

Comment:
this is counting excluded packages, so it contradicts the log message, see: OrtResult.getPackages(omitExcluded: Boolean = false)

Reply:
Phew, this was also broken for the `AnalyzerCommand`. Thanks, both fixed.